addon/components/item-picker/feature-service-preview/component.js

```diff
@@ -100,13 +99,13 @@ export default Component.extend({
    * ... we have an error
    * ... we need to choose a layer, and have not selected one
    */
-  isSelectDisabled: computed('forceLayerSelection', 'selectedLayer', 'isValidating', 'errorMessage.status', function () {
-    const errorMessage = this.get('errorMessage');
+  isSelectDisabled: computed('forceLayerSelection', 'selectedLayer', 'isValidating', 'validationResult.status', function () {
+    const validationResult = this.get('validationResult');
     let result = false;
     if (this.get('isValidating')) {
       result = true;
     }
-    if (errorMessage && errorMessage.status && errorMessage.status === 'error') {
+    if (validationResult && validationResult.status && validationResult.status === 'error') {
```

Comment:
Pretty sure this could be just `if (validationResult && validationResult.status === 'error') {`

If `validationResult.status` is `undefined` it will not `=== 'error`.